tests/func/test_commit.py

```diff
@@ -1,46 +1,48 @@
 from __future__ import unicode_literals
 
+import os
+
 import pytest
 
 from dvc.stage import StageCommitError
 from dvc.utils.stage import dump_stage_file
 from dvc.utils.stage import load_stage_file
 
 
-def test_commit_recursive(dvc_repo, repo_dir):
-    stages = dvc_repo.add(repo_dir.DATA_DIR, recursive=True, no_commit=True)
-    assert len(stages) == 2
-
-    assert dvc_repo.status() != {}
+def test_commit_recursive(tmp_dir, dvc):
+    tmp_dir.gen(
+        {"dir": {"file": "file content", "subdir": {"file2": "file2 content"}}}
+    )
+    stages = dvc.add("dir", recursive=True, no_commit=True)
 
-    dvc_repo.commit(repo_dir.DATA_DIR, recursive=True)
+    assert len(stages) == 2
+    assert dvc.status() != {}
 
-    assert dvc_repo.status() == {}
+    dvc.commit("dir", recursive=True)
+    assert dvc.status() == {}
 
 
-def test_commit_force(dvc_repo, repo_dir):
-    stages = dvc_repo.add(repo_dir.FOO, no_commit=True)
-    assert len(stages) == 1
-    stage = stages[0]
+def test_commit_force(tmp_dir, dvc):
+    tmp_dir.gen({"dir": {"file": "f content", "file2": "file2 content"}})
+    stage, = dvc.add("dir", no_commit=True)
 
-    with dvc_repo.state:
+    with dvc.state:
         assert stage.outs[0].changed_cache()
 
-    with open(repo_dir.FOO, "a") as fobj:
-        fobj.write(repo_dir.FOO_CONTENTS)
+    with open(os.path.join("dir", "file"), "a") as fobj:
+        fobj.write("modification")
```

Comment:
You can rewrite it with
```python
tmp_dir.gen("dir/file", "file content modified")  # or "text1 modified"
```
Since everything is literals, which we have right here, it is obvious anyways.